File: Employee.java
import java.util.ArrayList;

public class Employee {
    private String name;
    private String lastName;
    private ArrayList<BankAccount> accounts = new ArrayList<>();
    private ArrayList<BankAccount> invalidAccs = new ArrayList<>();
    private boolean f = false;

    public ArrayList<BankAccount> getInvalidAccs() {
        return invalidAccs;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public boolean getValidity(){
        return f;
    }

    public void setValidity(boolean f){
        this.f=f;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public ArrayList<BankAccount> getAccounts() {
        return accounts;
    }

    public void getFunds(int i) {
        System.out.println(accounts.get(i).getFunds());
    }

    public void showAccounts() {
        int i = 1;

        System.out.printf("| %-3s | %-4s | %-6s | %-8s | %-14s |%n", "#", "ID", "Type", "Status", "Funds");
        for (BankAccount toShow : accounts) {
            System.out.printf("| %-3s | %-4s | %-6s | %-8s | %-14s |%n", i,toShow.getAccNumber(), toShow.getAccType(), toShow.isValidAcc(), toShow.getFunds());
            i ++;
        }
    }

    
}

File: Employee.py
import BankAccount 

class Employee:
    __name = ""
    __last_name = ""
    __accounts = []
    __flag = False


    def __init__(self, name, last_name):
        self.__name = name
        self.__last_name = last_name


    def set_name (self, name):
        self.__name = name


    def set_last_name (self, last_name):
        self.__last_name = last_name


    def set_validity (self, flag):
        self.__flag = flag


    def set_account (self, account ):
        self.__accounts.append(account)
        self.__flag = True  


    def get_name (self):
        return self.__name


    def get_last_name (self):
        return self.__last_name


    def get_accounts (self):
        return self.__accounts
    

    def get_validity (self):
        return self.__flag
    

    
